### ISSUE TITLE
[ProgressView] Fix Voiceover announcements 

### ISSUE THREAD

**Issue by yarneo** [material-components/material-components-ios#4307]
Looks like there are cases where the progress view provides an announcement via voiceover, and this needs to be researched if we want that kind of behavior or we should change it.

---
#### Internal data
- Associated internal bug: [b/117178901](http://b/117178901)

**Comment by romoore** [comment]
A few things here:
1. Work with internal a11y team to ensure we're following best practices and get those documented.
2. In the mean time, make sure we're not announcing more often than UIProgressView.
3. If we're announcing less often than UIProgressView, leave it until the a11y team provides guidance.

**Comment by andrewoverton** [comment]
After some discussion in #4557 I don't believe this is an issue anymore.

**Comment by romoore** [comment]
Let's reopen until we've confirmed the behavior with the accessibility team and gotten recommendations published as mentioned in my comment above.

**Comment by andrewoverton** [comment]
@romoore, regarding [this comment](https://github.com/material-components/material-components-ios/issues/4307#issuecomment-395509968), I don't think we're announcing more than UIProgressView is, if by announcing we mean posting a `UIAccessibilityAnnouncementNotification`. We only do that when the `progress` property changes, and from what I can tell UIProgressView does the same thing--there is VoiceOver speech when you set the `progress` to a new value.

The change I suggested in #4557 was made because I thought the `UIAccessibilityLayoutChangedNotification` in `-setHidden:` would result in VoiceOver speech, and I noticed that UIProgressView didn't make any sound when `-setHidden:` was called on it.

Should I post a YAQ tagged with "accessibility" and "iOS" asking if our two UIAccessibility notifications are appropriate?
